VML/journey-driven-development
# Updating .gitignore, README and a few files. Added a build.sh file to…

… be used on Mac and Linux to validate pdf build.

## Synopsis of Changes

  - Added build script and instructions to allow writes to validate .pdf generation locally. 

## Proposed Changes

  - Added build.sh script to allow generation of .pdf file for local validation
  - Updated .gitignore to ensure Mac hidden files and *.pdf's are not added to repo
  - Updated readme.md with instructions on how to set up local for build validation
  - Updated readme.md with local build instructions
  - Added a few lines to some files to have them show in build properly
  
## Steps to Test
  
  - From your local repo
    - git pull
    - git checkout feature/localbuildsetup
    - sh build.sh
    - Validate Build Worked

- If it fails, ensure you have followed the instructions in the readme.md related to adding pandocs and latex to your local environment.
